portal-web/test/functional/com/liferay/portalweb/tests/enduser/collaboration/knowledgebase/KBArticle.testcase

```diff
@@ -731,6 +731,111 @@ definition {
 		AssetCategorization.viewTag(tagName = "Tag2");
 	}
 
+	@description = "This test ensures that the owner of the kb article receives a notification once the review date is reached."
+	@priority = 5
+	test CanReviewArticleWhenReviewDateIsReached {
+		property custom.properties = "company.default.time.zone=America/Los_Angeles${line.separator}feature.flag.LPS-165476=true";
+		property test.name.skip.portal.instance = "kbArticle#CanReviewArticleWhenReviewDateIsReached";
+
+		KBArticle.updateCheckInterval(fieldValue = 2);
+
+		KBArticle.openToAddKBArticle(
+			groupName = "Guest",
+			siteURLKey = "guest");
+
+		PortletEntry.inputTitle(title = "Knowledge Base Article Title");
+
+		CKEditor.addContent(content = "Knowledge Base Article Content");
+
+		KBArticle.setReviewDate(
+			enableReviewDate = "true",
+			increaseMinutes = 1);
+
+		PortletEntry.publish();
+
+		KBAdmin.openKBAdmin(siteURLKey = "guest");
```

Comment:
Please add as conditional to KBArticle.addCP. I asked Lily to do the same for expiration dates here https://github.com/liferay-lima/liferay-portal/pull/4017/commits/f5ab18ea135a95fa051aad378059b811ba0b68e2